ci/compose.sh

```diff
@@ -19,5 +19,5 @@ function compose_down_all {
 }
 
 function all_weaviate_ports {
-  echo "8080 8082 8083"
+  echo "8080 8081 8082 8083 8084"
```

Comment:
you need to use 8085 instead of 8084 as something else is running on there.
Also it would be great if you could use the same port numbers as the python client, just to keep it consistent.

Reply:
done